src/scripts/components/VideoForm.js

```diff
@@ -186,16 +127,21 @@ class VideoForm extends Component<Props, Object> {
     this.publishVideo(true)
   }
 
-  onPublishSubmit (e: Object) {
-    e.preventDefault()
-
-    this.props.showModal(
-      <ModalStake
-        videoId={this.state.id}
-        onSuccess={this.handlePublish}
-        user={this.props.user}
-      />
-    )
+  // onPublishSubmit (e: Object) {
```

Comment:
Delete this commented code?

Reply:
yes! deleted https://github.com/Paratii-Video/paratii-portal/pull/301/commits/ef56e9a93bb88ac25241e85ce446283d1bd98d44